packages/drivers/driver-web-cache/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@fluidframework/driver-web-cache",
+  "version": "0.58.1000",
```

```suggestion
  "version": "0.58.2000",
```
these were bumped during the course of this review